lib/internal/net.js

```diff
@@ -67,7 +67,7 @@ function makeSyncWrite(fd) {
 }
 
 module.exports = {
-  kWrapConnectedHandle: Symbol('wrapConnectedHandle'),
+  kReinitializeHandle: Symbol('reinitializeHandle'),
```

Comment:
nit:
```suggestion
  kReinitializeHandle: Symbol('kReinitializeHandle'),
```

Reply:
Sorry, I missed this - Included in #46790.